docs/querying/functions.md

```diff
@@ -317,6 +317,12 @@ bound of that bucket is greater than
 bucket. Otherwise, the upper bound of the lowest bucket is returned for
 quantiles located in the lowest bucket. 
 
+You can use `histogram_quantile(0, v instant-vector)` to get the estimated minimum value stored in
+a native histogram.
```

Comment:
The same works for a classic histogram (but of course which a usually much lower accuracy). I would just delete "native" here.